tests/reference/asr-arrays_02-da94458.stderr

```diff
@@ -0,0 +1,5 @@
+semantic error: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
+  --> tests/errors/arrays_02.py:28:8
+   |
+28 | assert r1.a == t1.a
+   |        ^^^^^^^^^^^^ 
```

Comment:
And this:
```
help: use any(r1.a == t1.a) or all(r1.a == t1.a)
```

Reply:
>help: use any(r1.a == t1.a) or all(r1.a == t1.a)

Here, we do not have access to the string `r1.a == t1.a`. When printing diagnostics the input file is read (from disk) and the error line/string is obtained form the read source input.

Shall we do the following?
```
help: use any() or all()
```
Or shall we keep the message as it is currently in the main branch?

Reply:
We have access to AST and partial ASR, so we should be able to construct `t1`, `r1` and `a`. And we know it's a struct access and we know it's a comparison. So you use this information to create the text `any(r1.a == t1.a)`.

Reply:
I made this https://github.com/lcompilers/lpython/issues/2159, so that we don't forget.